.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Upload website to QA bucket
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: 
+      - master
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --acl public-read --follow-symlinks --delete
+      env:
+        AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
```

Comment:
Don't think this needs to be a secret.

Reply:
You're probably right, I can make a follow-up PR to remove it if we think it'll make things easier